packages/http/yarn.lock

```diff
@@ -2,13 +2,6 @@
 # yarn lockfile v1
 
 
-"@helium/currency@^3.25.1":
```

Comment:
also wondering why the lock file changed for this 🤔

Reply:
I didn't even notice this and not sure why it would have changed it. Must be my lack of js knowledge showing. Gonna clean these two things up though.